### ISSUE TITLE
Tang-Nano_Tetris/doc/BlockDiagram.png from wrong repository

### ISSUE THREAD

**Issue by charkster** [kingyoPiyo/Tang-Nano_Tetris#1]
Looks like Tang-Nano_Tetris/doc/BlockDiagram.png is from your Tang-Nano_100BASE-FX_Talker repository. Do you want to remove it?
Tang-Nano_Tetris/doc/img/BlockDiagram.png looks correct.
What program do you use to view CE3 files?
Can I use this controller?
https://www.amazon.com/Hyperkin-Scout-Premium-Controller-super-nintendo/dp/B0771371PM
Is this diagram correct?
https://www.igorkromin.net/fp-content/images/snestones/NesSnesPinout.png
Which LCD are you using?
Thank you.

**Comment by charkster** [comment]
@kingyoPiyo I saw that you are using a ATM0430D25 LCD in your Tang-Nano_PartyParrot
Could I use this one?
https://www.digikey.com/en/products/detail/az-displays/ATM0430D19A/12699402
Thank you.

**Comment by kingyoPiyo** [comment]
Hello,
Thanks for the post.
Tang-Nano_Tetris/doc/BlockDiagram.png is wrong and has been removed. The others are correct.

I use the following schematic editor to open the CE3 file. Sorry, it is for Japanese only.
https://www.suigyodo.com/online/schsoft.htm

The SNES controller you provided seems to be fine. The pin assignments are also correct.

Regarding the display, the ATM0430D19A cannot be used because of the different pin layout.
You may be able to use the following.(There is no confirmation)
DLC0430BBM07RT-3
DLC0430BCP10RF-2
HT043DA-V.0
FG040346DSSWBG03

**Comment by charkster** [comment]
@kingyoPiyo I found the english version for BSch3V https://www.suigyodo.com/online/e/index.htm
I checked the prices of your suggested LCDs, ~~and I was thinking that I could try the ATM0430D19A with changing your fpga_project.cst file for the new pin assignments.~~ I guess I have to use a 40 pin cable to fit the FPGA board.
Thanks for checking the SNES pins, it's my first time using them.
I'll let you know when I have this working, I am hoping to have the total cost of FPGA board, controller and LCD under $30 (not including shipping).
